### ISSUE TITLE
package version shown as part of the package name in openSUSE

### ISSUE THREAD

**Issue by cboltz** [openmainframeproject/software-discovery-tool#155]
If you search for openSUSE packages, you'll get results like

```
Package Name              Version      Distro
apparmor-parser-3.0.1     9.1          OpenSUSE (Tumbleweed)
```

Note that the package name contains the version, and the version column contains only the release.

The actual package name is something like `apparmor-parser-3.0.1-9.1.s390x.rpm`.

The pattern for openSUSE package names is `$packagename-$version-$release.$arch.rpm`, where `$version` is the upstream version number, and `$release` shows (first number) how often a package was touched (for example adding a patch), and (second number) how often it was rebuilt.

In this case, I'd expect the package name to be `apparmor-parser`, and the version `3.0.1` (just the upstream release number) or `3.0.1-9.1` (both upstream release number and openSUSE release number).

**Comment by pleia2** [comment]
We're going to tackle this along with our fix for #145, so we won't deploy new data sources until we're sure about correcting the version number situation. Thanks for chiming in with your expertise here to share how to parse the package name, it's a huge help!

**Comment by Princee215** [comment]
Thanks, @cboltz, this is very helpful. Me and @arshPratap were thinking in the same way of splitting the name into package name and version. Though needed some proof of what we do is the right way to do so.
We'll work on this and follow the convention and will soon revert back soon.
